frontend/lib/api/form/pollingstation/pollingStationUtils.ts

```diff
@@ -476,4 +476,11 @@ export function updateFormStateAfterSubmit(
       section.warnings = section.warnings.filter((err) => !isGlobalValidationResult(err));
     });
   }
+
+  // Reset acceptWarnings when a page gets an error or has no warnings anymore
+  Object.values(formState.sections).forEach((section) => {
+    if (section.acceptWarnings && (section.errors.length > 0 || section.warnings.length === 0)) {
```

Comment:
Could there be a scenario that there is one section without warnings and one with warnings? Is the expected behaviour that the `acceptWarnings` will be reset in this case?

Reply:
Yes, that is possible, so that's why we only reset the acceptWarnings for the section it is applicable for. We save this variable for every section individually.

Reply:
Ah this is for one section, got it, thanks!